<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWidgetPedalRobotNVHGetSpeed</class>
 <widget class="QWidget" name="SettingWidgetPedalRobotNVHGetSpeed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_prg">
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="title">
      <string>程序</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1">
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>开度-速度测试</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBox_o2v">
          <item>
           <property name="text">
            <string>True</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>False</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>设备串口号</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_serialDevice">
          <item>
           <property name="text">
            <string>COM1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM9</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>控制方式</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBox_loop">
          <item>
           <property name="text">
            <string>开环</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>闭环</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_pulse">
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="title">
      <string>脉冲转换</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,1">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>脉冲转换车速</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_selectLab"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>脉冲(Hz)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>车速(km/h)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_calibratePulseFrequency"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_calibrateSpeed"/>
        </item>
        <item row="0" column="2" rowspan="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
